setup

```diff
@@ -303,15 +332,20 @@ case "$1" in
 
     if [ "$SYSTEMCTL_SKIP" == "N" ]; then
 	    systemctl daemon-reload
-	    systemctl start qm.service
-            if [ "$(systemctl is-active qm.service)" != "active" ]; then
-               journal=$(journalctl --no-pager -xu qm.service)
-               echo QM service is not up details:
-               echo "$journal"
-               exit 1
-            fi
+        systemctl start qm.service || {
+            logger "ERROR" "'systemctl start qm.service' has failed.";
+            systemctl status --no-pager -l qm.service;
```

Comment:
Please use 
journalctl --no-pager -xu  qm.service
Once setup script is failing in upstream gate, most iunfo could be read from logs

As you are doing later